src/scripts/domParsers/videoUtils.ts

```diff
@@ -65,6 +67,23 @@ export module VideoUtils {
 		return values;
 	}
 
+	export function getKhanAcademyVideoSrcValue(pageContent: string): string {
```

Comment:
I realize that we're not consistent through the codebase, but we should use method header comments to at least keep this file looking good. :-)

Note that this is true for the other two new functions below as well.